src/utils.ts

```diff
@@ -154,6 +153,13 @@ export async function cancellableExtract(
     throw new CancellableExtractFailedError(targetDir, outputDir);
   }
 }
+export function range(start: number, end: number) {
```

Comment:
I guess it's also unnecessary. (if we need, [`Array.from()`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/from) can help)

Reply:
This is a python-like `range` for generating numbers. `Array.from()` is converted to an array from an array-like object. Is there a way to generate numbers to use `Array.from()`?

Reply:
> Is there a way to generate numbers to use Array.from()?

Yes, we can. (copied from Mozilla's example)

```js
// Generate a sequence of numbers
// Since the array is initialized with `undefined` on each position,
// the value of `v` below will be `undefined`
Array.from({length: 5}, (v, i) => i);
// [0, 1, 2, 3, 4]
```

Reply:
ac4a3f4 may apply this idea